crane/cmd.go

```diff
@@ -252,6 +262,11 @@ See the corresponding docker commands for more information.`,
 	cmdLogs.Flags().StringVarP(&options.tail, "tail", "", "all", "Output the specified number of lines at the end of logs")
 	cmdLogs.Flags().BoolVarP(&options.colorize, "colorize", "z", false, "Output the lines with one color per container")
 
+	cmdTmpl.Flags().StringVarP(&options.tmplInput, "tmpl", "t", "", "The input template file")
+	cmdTmpl.Flags().StringVarP(&options.tmplOutput, "output", "o", "", "The output file or directory (os.Stdout as default)")
```

Comment:
Oh, ignore that, sorry. It's because previously I thought in indicating a directory to output every file with the name of each container, but then I decided to use a string format with `%s` to be able to adding a file extension. I will check every text, I only was waiting to the PR is approved.